caster/apps/atom.py

```diff
@@ -3,462 +3,408 @@
+            ACP("Split Into Lines", "Atom: Split Into lines"),
         "single section":
-            R(Key("escape"), rdescript="Atom: Single Section"),
+            ACK("escape", "Atom: Single Section"),
         "select [to] top":
-            R(Key("cs-home"), rdescript="Atom: Select to Top"),
+            ACK("cs-home", "Atom: Select to Top"),
         "select [to] bottom":
-            R(Key("cs-end"), rdescript="Atom: Select to Bottom"),
-        #"select line":                             R(Key("c-l"), rdescript="Atom: Select Line"),
-        #"select word [<n>]":                       R(palettized("Editor: Word"), rdescript="Atom: Select Word") * Repeat(extra="n"),
-        "[select] [to] begin [of] word [<n>]":
-            R(Key("cs-left"), rdescript="Atom: Select to Beginning of Word #")*
-            Repeat(extra="n"),
-        "[select] [to] end word [<n>]":
-            R(Key("cs-right"), rdescript="Atom: Select to End of Word #")*
-            Repeat(extra="n"),
+            ACK("cs-end", "Atom: Select to Bottom"),
+        #"select line":                             ACK("c-l", "Atom: Select Line"),
+        #"select word [<n>]":                       ACP("Editor: Word", "Atom: Select Word") * Repeat(extra="n"),
         "[select] [to] begin line":
-            R(palettized("Editor: Select to Beginning of Line"),
-              rdescript="Atom: Select to Beginning of line"),
-        "[select] [to] first character":
-            R(Key("s-home"), rdescript="Atom: Select to First Character of Line"),
-        "[select] [to] end line":
-            R(Key("s-end"), rdescript="Atom: Select to End of line"),
+            ACP("Editor: Select to Beginning of Line",
+                "Atom: Select to Beginning of line"),
         "[select] inside brackets":
-            R(Key("ac-m"), rdescript="Atom: Select Inside Brackets"),
+            ACK("ac-m", "Atom: Select Inside Brackets"),
         #Find Menu
-        #"find in buffer":                          R(Key("c-f"), rdescript="Atom: Find in Buffer"),
-        "replacing in buffer":
-            R(Key("ac-f"), rdescript="Atom: Replacing in Buffer"),
+        "(replace | replacing) in buffer":
+            ACK("ac-f", "Atom: Replacing in Buffer"),
         "select next":
-            R(Key("a-f3"), rdescript="Atom: Select All"),
+            ACK("a-f3", "Atom: Select All"),
         "find replace next":
-            R(palettized("Find and Replace: Replace Next"),
-              rdescript="Atom: Replace Next"),
+            ACP("Find and Replace: Replace Next", "Atom: Replace Next"),
         "find replace all":
-            R(palettized("Find and Replace: Replace All"), rdescript="Atom: Replace All"),
+            ACP("Find and Replace: Replace All", "Atom: Replace All"),
         "find buffer":
-            R(Key("c-b"), rdescript="Atom: Find Buffer"),
+            ACK("c-b", "Atom: Find Buffer"),
         "find file":
```

Comment:
Perhaps "(find | go to) file" to be consistent with the other editors.

Reply:
I'll fix it, but it should eventually be handled by #211.